src/clj_gatling/reporters/raw_reporter.clj

```diff
@@ -19,17 +19,21 @@
    :collector 'clj-gatling.reporters.raw-reporter/collector
    :generator 'clj-gatling.reporters.raw-reporter/generator})
 
-(defn- write-ends-as-lines [file-name lines]
+(defn- write-edns-as-lines [file-name lines]
   (with-open [wtr (BufferedWriter. (FileWriter. file-name))]
     (loop [lines-left lines]
-      (let [line (pr-str (first lines-left))]
-        (.write wtr line)
+      (let [line         (first lines-left)
+            cleaned-line (update line :requests (fn [reqs]
+                                                  (map #(if (:exception %)
+                                                          (update % :exception str)
+                                                          %) reqs)))]
```

Comment:
We clean any returned exceptions in the file-based raw reporter by stringifying them. This loses the stacktrace, but allows the log to be parsed correctly. Arguably we already know the source because it's associated with a specific step, and each step is probably quite a small number of lines of code.